packages/manager/modules/bm-server-components/src/server-dashboard/dashboard.html

```diff
@@ -40,6 +40,9 @@
             data-traffic-information="$ctrl.trafficInformation"
             data-dedicated-server="$ctrl.dedicatedServer"
             data-current-active-link="$ctrl.currentActiveLink"
+            data-state-prefix="$ctrl.statePrefix"
```

Comment:
still necessary ?

Reply:
Yes for consumption tile where there is a link to other tabs (that not child).